Packs/CloudIncidentResponse/Playbooks/playbook-Cortex_XDR_-_XCloud_Token_Theft_Response.yml

```diff
@@ -0,0 +1,2293 @@
+        complex:
+          root: incident.xdralerts
+          accessor: cloudprovider
+          transformers:
+          - operator: uniq
+      instanceName:
+        complex:
+          root: CloudIndicators
+          accessor: resource_name
+      username:
+        complex:
+          root: CloudIndicators
+          accessor: username
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 240,
+          "y": 2050
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "49":
+    id: "49"
+    taskid: 28affdfa-fc9e-450b-8148-ed1f29bfa1f4
+    type: condition
+    task:
+      id: 28affdfa-fc9e-450b-8148-ed1f29bfa1f4
+      version: -1
+      name: Investigate and set verdict
+      description: You should investigate the data collected manually and choose how the playbook should continue.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "61"
+      Malicious:
+      - "50"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -230,
+          "y": 920
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "50":
+    id: "50"
+    taskid: c0f0c235-a092-4f49-81d6-642230c1e69a
+    type: condition
+    task:
+      id: c0f0c235-a092-4f49-81d6-642230c1e69a
+      version: -1
+      name: Should execute early containment?
+      description: Whether to execute early containment and block the IP address and respond to the username involved.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "5"
+      "yes":
+      - "11"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: inputs.earlyContainment
+            iscontext: true
+          right:
+            value:
+              simple: "True"
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1100
+        }
+      }
+    note: false
+    timertriggers:
+    - fieldname: containmentsla
+      action: start
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "51":
+    id: "51"
+    taskid: 7e6cfe98-fc45-421f-83a9-54b233392498
+    type: playbook
+    task:
+      id: 7e6cfe98-fc45-421f-83a9-54b233392498
+      version: -1
+      name: Cloud Response - Generic
```

Comment:
we should let the customer decide to run the containment (+ everything should be manual, like block IOC) also for eradication plan

Reply:
added the auto*remediation fields to the playbook inputs.